compositor_render/src/renderer.rs

```diff
@@ -1,16 +1,19 @@
-use std::{collections::HashMap, rc::Rc};
+use std::sync::Arc;
 
 use compositor_common::{scene::Scene, Frame};
 
-use crate::registry::{self, TransformationRegistry};
+use crate::{
+    input_frames::InputFrames,
+    registry::{self, TransformationRegistry},
+};
 
 use self::transformation::Transformation;
 
 pub mod texture;
 pub mod transformation;
 
 pub struct Renderer {
-    wgpu_ctx: Rc<WgpuCtx>,
+    wgpu_ctx: Arc<WgpuCtx>,
```

Comment:
I don't see why this is necessary

Reply:
It was useful when I wanted to add `Renderer` directly to the pipeline, but since it's not done here in that way, we can keep it as `Rc`.